azerothcore/azerothcore-wotlk
# feat(Core): Hide Quest Sparkle Config Option

## Changes Proposed:
-  Adds a config option to allow server owner to disable the sparkle of quest items (sparkle is enabled by default).
-  This was the behavior until patch 2.3, which added the sparkle. Some players may prefer to not have it.
-  From online comments I read, it seems this is a client-side option in current retail WoW

## Issues Addressed:

- Closes unreported issue

## SOURCE:

Patch 2.3

## Tests Performed:

- Tested in game
- Objects sparkle with set to true, objects do not sparkle when set to false
- Objects can be used correctly and have correct interaction regardless of flag setting


## How to Test the Changes:


1. Check that quest objects sparkle when option is turned on
2. Check that quest objects do not sparkle when option is turned off

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
